frontend/express/public/javascripts/countly/vue/templates/sidebar/analytics-menu.html

```diff
@@ -1,31 +1,21 @@
 <div class="cly-vue-sidebar__menu">
-    <div class="cly-vue-sidebar__header">
-        <cly-select-x
-            search-placeholder=""
-            placeholder="Select App"
-            v-model="selectedApp"
-            :mode="selectMode"
-            :options="allApps"
-            @change="onChange"
-            popClass="cly-vue-sidebar__analytics-popper"
-            prefixIcon="el-icon-platform-eleme"
-            label="web"
-            :width="200">
-            <template v-slot:trigger="slotProps">
-                <el-input
-                    v-model="activeApp.name"
-                    readonly="readonly"
-                    clearable>
-                    <template slot="prefix">
-                        <div class="cly-vue-sidebar__app-image" :style="{backgroundImage: 'url(' + activeApp.image + ')'}"></div>
-                    </template>
-                    <template slot="suffix">
-                        <i class="el-select__caret el-input__icon" :class="['el-icon-' + suffixIconClass(slotProps.dropdown)]"></i>
-                    </template>
-                </el-input>
-            </template>
-        </cly-select-x>
+    <div class="cly-vue-sidebar__header cursor-pointer bu-m-3 bu-mb-4" @click="toggleAppSelection()">
+        <div class="cly-vue-sidebar__app-selector bu-is-flex">
```

Comment:
**toggleAppSelection()** should be added to the **cly-vue-sidebar__app-selector** div.
and same for **cursor-pointer** class.